pkg/agent/core/metrics.go

```diff
@@ -31,7 +35,8 @@ type LFCMetrics struct {
 	CacheMissesTotal float64
 	CacheWritesTotal float64
 
-	ApproximateWorkingSetSizeTotal float64 // approximate_working_set_size
+	// lfc_approximate_working_set_size_seconds
```

Comment:
These are indexed by number of minutes?

Reply:
Yeah, currently the exposed metric name is `lfc_approximate_working_set_size_seconds`.

Although that metric _also_ is labeled with `duration_minutes`. Perhaps something there should change -- I'd be interested in your thoughts re: neondatabase/neon#8298

Reply:
> Perhaps something there should change

Like what?

One thing metric `..._seconds` being labeled with `duration_minutes` is no ideal. See also https://github.com/neondatabase/neon/pull/8298#discussion_r1680794911.

Reply:
Fixed in 496a333a5aa4452f7cbb1230ca7f3ca3f65ff9f0